website/tosti/templates/tosti/base.html

```diff
@@ -6,6 +6,9 @@
     <title>
         {% block title %}TOSTI{% endblock %}
     </title>
+    <script>
+        CSRF_TOKEN = "{{ csrf_token }}";
```

Comment:
this is not required, your js can just access `Cookies.get('csrftoken')` I think (at least that is what concrexit does)